api/src/opentrons/hardware_control/ot3api.py

```diff
@@ -1842,6 +1842,8 @@ async def dispense(
             raise
         else:
             dispense_spec.instr.remove_current_volume(dispense_spec.volume)
+            if push_out or dispense_spec.instr.push_out_volume:
```

Comment:
probably better for this to just be this since it's what actually controls things
```suggestion
            if dispense_spec.instr.push_out_volume:
```

Reply:
if caller sets `push_out=0` then the pipette is still ready-to-aspirate, so we need to either check this argument or we could specify what push-out volume was used within the `dispense_spec` and check that instead